main.nf

```diff
@@ -382,6 +411,110 @@ process generate_peptides {
     """
 }
 
+/*
+ * Split peptides into chunks of peptides that are to be predicted against the
+ * same allele for parallel prediction
+ */
+process split_peptides {
+    input:
+    path  microbiomes           from  ch_microbiomes_table
+    path  conditions            from  ch_condition_microbiome_table
+    path  conditions_alleles    from  ch_condition_allele_table
+    path  alleles               from  ch_alleles_table
+    path  peptides              from  ch_peptides
+    path  microbiomes_proteins  from  ch_proteins_microbiomes
+    path  proteins_peptides     from  ch_proteins_peptides
+
```

Comment:
Could you add a comment here, that all these tables are only used to get from `peptide_id` to `allele_name`?